src/main/java/Whoriskey/Leigh/React_Spring_api/controller/EligibilityController.java

```diff
@@ -0,0 +1,64 @@
+package Whoriskey.Leigh.React_Spring_api.controller;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import Whoriskey.Leigh.React_Spring_api.model.Eligibility;
```

Comment:
Is the full name required or is this like a package name? If it is something that is re-usable or a package then maybe give it a better naming convention?